openmc/material.py

```diff
@@ -908,22 +899,46 @@ def get_nuclide_atom_densities(self):
 
         return nuclides
 
-    def get_nuclide_activity(self):
-        """Return activity in [Bq] for each nuclide in the material
+    def get_activity(self, units: str = 'Bq/cm3', by_nuclide: bool = False):
+        """Returns the activity of the material or for each nuclide in the
+        material in units of [Bq], [Bq/g] or [Bq/cm3].
 
         .. versionadded:: 0.13.1
 
+        Parameters
+        ----------
+        units : {'Bq', 'Bq/g', 'Bq/cm3'}
+            Specifies the type of activity to return, options include total
+            activity [Bq], specific [Bq/g] or volumetric activity [Bq/cm3].
+            Default is volumetric activity [Bq/cm3].
+        by_nuclide : bool
+            Specifies if the activity should be returned for the material as a
+            whole or per nuclide. Default is False.
+
         Returns
         -------
-        dict
-            Dictionary whose keys are nuclide names and values are activity in
-            [Bq].
+        Union[dict, float]
+            If by_nuclide is True then a dictionary whose keys are nuclide 
+            names and values are activity is returned. Otherwise the activity
+            of the material is returned as a float.
         """
+
+        cv.check_value('units', units, {'Bq', 'Bq/g', 'Bq/cm3'})
+        cv.check_type('by_nuclide', by_nuclide, bool)
+
+        if units == 'Bq':
+            multiplier = self.volume
+        elif units == 'Bq/g':
+            multiplier = 1
+        elif units == 'Bq/cm3':
```

Comment:
```suggestion
        elif units == 'Bq/cm3':
            multiplier = 1
        elif units == 'Bq/g':
```
These got flipped somewhere along the way and cause the tests to fail